app/org/sagebionetworks/bridge/BridgeUtils.java

```diff
@@ -338,5 +340,36 @@ public static String encodeURIComponent(String component) {
         }
         return encoded;
     }
+    
+    public static String passwordPolicyDescription(PasswordPolicy policy) {
+        StringBuilder sb = new StringBuilder();
+        sb.append("Password must be ").append(policy.getMinLength()).append(" or more characters");
+        if (policy.isLowerCaseRequired() || policy.isNumericRequired() || policy.isSymbolRequired() || policy.isUpperCaseRequired()) {
+            sb.append(", and must contain at least ");
+            List<String> phrases = new ArrayList<>();
+            if (policy.isLowerCaseRequired()) {
+                phrases.add("one lower-case letter");
+            }
+            if (policy.isUpperCaseRequired()) {
+                phrases.add("one upper-case letter");
+            }
+            if (policy.isNumericRequired()) {
+                phrases.add("one number");
+            }
+            if (policy.isSymbolRequired()) {
+                phrases.add("one symbolic character");
```

Comment:
It's a very ugly list and not very user friendly to read. But maybe I can give examples or something.